apps/src/lib/kits/maker/ExternalButton.js

```diff
@@ -0,0 +1,32 @@
+import {EXTERNAL_PINS as MB_EXTERNAL_PINS} from './MicroBitConstants';
+import {EventEmitter} from 'events';
+
+export default function ExternalButton(board) {
+  // There are six button events, ['', 'down', 'up', 'click', 'long-click', 'hold']
+  this.buttonEvents = new Array(6).fill(0);
+  this.board = board;
+  this.pullup = MB_EXTERNAL_PINS.includes(this.board.pin);
+  if (this.pullup) {
+    this.board.mb.trackDigitalPin(this.board.pin, 1);
+  }
+  this.connect = false;
```

Comment:
What does this represent?

Reply:
Good catch - it's a remnant of an old implementation strategy. Will remove.